README.md

```diff
@@ -1,2 +1,48 @@
 # FIRE-WiFiCalling
 WiFi Calling App to be used with LibreServer
+
+## Environment configuration
+
+To configure basic dev environment
+
+First of all create an account in https://apps.ionic.io
+
+```
+sudo npm install -g ionic cordova
```

Comment:
mmm, strange...it did install 2.2.3 on my machine. I guess it has to do with Ubuntu repos vs your Debian ones. Let me just correct it. Yours is more precise than mine and will end up with fewer problems for the users.